docs/writing-policies/typescript.md

```diff
@@ -1,41 +1,42 @@
 ---
-sidebar_label: "TypeScript"
-title: ""
+sidebar_label: Typescript
+title: Typescript
+description: Writing Kubewarden policies with Typescript
+keywords: [at least one]
+doc-type: [one of how-to, explanation, tutorial, reference]
+doc-topic: [root-branch]
+doc-persona: [default]
 ---
 
-# TypeScript
-
 As stated on the [official website](https://www.typescriptlang.org/):
 
 > TypeScript extends JavaScript by adding types.
 >
 > By understanding JavaScript, TypeScript saves you time catching errors and
 > providing fixes before you run code.
 
-TypeScript **cannot** be converted to WebAssembly, however
-[AssemblyScript](https://www.assemblyscript.org/) is a **subset** of TypeScript
-designed explicitly for WebAssembly.
+TypeScript can't target WebAssembly, however
+[AssemblyScript](https://www.assemblyscript.org/)
+is a **subset** of TypeScript designed for WebAssembly.
 
-## Current State
+## Current state
 
-Currently there's no Kubewarden SDK for AssemblyScript. We haven't created it
-because of lack of time. We will do that in the near future.
+Currently, there's no Kubewarden SDK for AssemblyScript.
+Resources permitting, we hope to have on in the near future.
 
-In the meantime, there seem to be some limitatations affecting AssemblyScript:
+However, there are limitations affecting AssemblyScript:
 
 * There's no built-in way to serialize and deserialize Classes to
-  and from JSON. See [this issue](https://github.com/AssemblyScript/assemblyscript/issues/292)
+  and from JSON.
+  See [this issue](https://github.com/AssemblyScript/assemblyscript/issues/292)
 * It *seems* there's no JSON path library for AssemblyScript
 
 ## Example
 
 [This GitHub repository](https://github.com/kubewarden/pod-privileged-policy)
```

Comment:
```suggestion
[This GitHub repository branch ](https://github.com/kubewarden/pod-privileged-policy/tree/assemblyscript-implementation)
```

We rewrote the policy in Rust, but the old implementation is in that branch.

Reply:
Thanks.